cmd/mtail/main.go

```diff
@@ -62,7 +62,8 @@ var (
 	logRuntimeErrors     = flag.Bool("vm_logs_runtime_errors", true, "Enables logging of runtime errors to the standard log.  Set to false to only have the errors printed to the HTTP console.")
 
 	// Ops flags.
-	pollInterval                = flag.Duration("poll_interval", 250*time.Millisecond, "Set the interval to poll all log files for data; must be positive, or zero to disable polling.  With polling mode, only the files found at mtail startup will be polled.")
+	pollInterval                = flag.Duration("poll_interval", 250*time.Millisecond, "Set the interval to poll all log files for list; must be positive, or zero to disable polling.  With polling mode, only the files found at mtail startup will be polled.")
+	pollLogInterval             = flag.Duration("poll_log_interval", 250*time.Millisecond, "Set the interval to poll all log files for data; must be positive, or zero to disable polling.  With polling mode, only the files found at mtail startup will be polled.")
```

Comment:
I'd like these inverted; the original flag should remain for the log stream poll interval, and the new flag for the pattern/list interval please.

The help string for both need to be updated as well.

Reply:
ok, i'd invert them